base/markdown/render/rich.jl

```diff
@@ -3,7 +3,7 @@ function tohtml(io::IO, m::MIME"text/html", x)
 end
 
 function tohtml(io::IO, m::MIME"text/plain", x)
-    writemime(io, m, x)
+    printesc(io, sprint(writemime, m, x))
```

Comment:
@one-more-minute sorry, I still don't understand the fix you're eluding to here/below!